lib/exercism/five_a_day_count.rb

```diff
@@ -0,0 +1,42 @@
+class FiveADayCount < ActiveRecord::Base
+  belongs_to :user
+  # has_and_belongs_to_many :user_exercise
+  #
+  # scope :today, lambda { where(day: Date.today) }
+
```

Comment:
Yeah, I think this happened during the merge/squash weird mess. There are other things we are finding that are not supposed to be there.